data/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Correct -- it would also suffice to create the directory in `load_hca_metadata.py`, but I kind of like the idea of "this directory is a standardized part of our application, but you should never commit the contents". I'm quite happy ignoring the directory entirely though.